doc/neo-tree.txt

```diff
@@ -198,6 +198,13 @@ s             = open_vsplit: Same as open, but opens in a vertical split.
 
 t             = open_tabnew: Same as open, but opens in a new tab.
 
+                  open_drop: Same as open, but opens with the `:drop` command. 
```

Comment:
Since it is a built-in help topic, you can actually just wrap it in `|` and it will become a hyperlink to that help topic:
```suggestion
                  open_drop: Same as open, but opens with the |:drop| command. 
```

Reply:
Ahaa! I am clearly a novice! Good call: https://github.com/rkoval/neo-tree.nvim/commit/82cd313328ebfaf42409611a739d1ba5f8467173